2_process/src/summarize_targets.R

```diff
@@ -1,5 +1,5 @@
-summarize_targets <- function(ind_file, ...) {
-  ind_tbl <- tar_meta(c(...)) %>%
+summarize_targets <- function(ind_file, v) {
```

Comment:
We typically try to name function parameters so that it's clear to what they refer. Here, `v` is a pretty ambiguous argument name -- could you change it to something that better captures the information that is being passed to this function?